src/mono/netcore/sample/HelloWorld/Program.cs

```diff
@@ -7,10 +10,9 @@ internal class Program
         private static void Main(string[] args)
         {
             bool isMono = typeof(object).Assembly.GetType("Mono.RuntimeStructs") != null;
+            Console.WriteLine();
             Console.WriteLine("Hello World " + (isMono ? "from Mono!" : "from CoreCLR!"));
-            Console.WriteLine(typeof(object).Assembly.FullName);
-            Console.WriteLine(System.Reflection.Assembly.GetEntryAssembly ());
-            Console.WriteLine(System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription);
```

Comment:
What's the motivation behind getting rid of this?

Reply:
ah, actually an unnecessary change